pkg/scheduler/framework/plugins/podtopologyspread/filtering.go

```diff
@@ -270,11 +270,12 @@ func (pl *PodTopologySpread) calPreFilterState(ctx context.Context, pod *v1.Pod)
 
 	tpCountsByNode := make([]map[topologyPair]int, len(allNodes))
 	requiredNodeAffinity := nodeaffinity.GetRequiredNodeAffinity(pod)
+	logger := klog.FromContext(ctx)
 	processNode := func(i int) {
 		nodeInfo := allNodes[i]
 		node := nodeInfo.Node()
 		if node == nil {
-			klog.ErrorS(nil, "Node not found")
+			logger.Error(nil, "Node not found")
```

Comment:
We added this log line several releases ago when it was possible for us to have nil nodes in the cache.

We no longer have those. We can remove this log line to avoid the `FromContext`